### DISCUSSION TITLE
Gene expression visualization

### DISCUSSION THREAD

**Discussion by adrdufour** [GreenleafLab/ArchR#1644] [Questions / Documentation] [answered]
Hello,

I would like to know how to visualize gene expression from my scRNAseq (I analyse a 10x multiome scRNAseq + scATACseq)

I have used the plot embedding commands 

```
plotEmbedding(
    ArchRProj = archrproj, 
    colorBy = "GeneScoreMatrix", 
    name = "NANOG", 
    embedding = "UMAP_Harmony",
    imputeWeights = getImputeWeights(archrproj)
)
```

But I don't know what genescorematrix refers to 

Thanks for your help

**Comment by rcorces** [comment, marked as answer]
Multiome tutorial currently hosted here though it will soon be updated in the manual:
https://greenleaflab.github.io/ArchR_2020/Ex-Analyze-Multiome.html

You want "GeneExpressionMatrix" if you are using multiome data.

**Comment by adrdufour** [comment]
Thanks for your responses, I have a few other questions: 
Is it possible to see genes linked to motifs after addMotifAnnotations

I would also know if it's possible to plot gene expression per cluster alongside Normalized signal with plotBrowserTrack

**Reply by rcorces** [reply]
> Is it possible to see genes linked to motifs after addMotifAnnotations

Yes - this information is stored in the file indicated by `$Matches` in you annotation. This would be stored as a binary matrix of motif presence per feature (i.e per peak). You would have to do the work to associate those peaks with genes.
```
> projHeme5@peakAnnotation$Motif
$Name
[1] "Motif"

$motifs
PWMatrixList of length 870
names(870): TFAP2B_1 TFAP2D_2 TFAP2C_3 TFAP2E_4 TFAP2A_5 ... SMAD9_867 SOX6_868 TBX18_869 TBX22_870

$motifSummary
DataFrame with 870 rows and 3 columns
                 name                     ID      strand
          <character>            <character> <character>
TFAP2B_1       TFAP2B ENSG00000008196_LINE..           *
TFAP2D_2       TFAP2D ENSG00000008197_LINE..           *
TFAP2C_3       TFAP2C ENSG00000087510_LINE..           *
TFAP2E_4       TFAP2E ENSG00000116819_LINE..           *
TFAP2A_5       TFAP2A ENSG00000137203_LINE..           *
...               ...                    ...         ...
SMAD5_866       SMAD5 ENSG00000113658_LINE..           *
SMAD9_867       SMAD9 ENSG00000120693_LINE..           *
SOX6_868         SOX6 ENSG00000110693_LINE..           *
TBX18_869       TBX18 ENSG00000112837_LINE..           *
TBX22_870       TBX22 ENSG00000122145_LINE..           *

$Positions
[1] "/corces/home/rcorces/ArchR/tutorial_201127/Save-ProjHeme5/Annotations/Motif-Positions-In-Peaks.rds"

$Matches
[1] "/corces/home/rcorces/ArchR/tutorial_201127/Save-ProjHeme5/Annotations/Motif-Matches-In-Peaks.rds"
```


> I would also know if it's possible to plot gene expression per cluster alongside Normalized signal with plotBrowserTrack

Not yet
